components/loader-alt/types.ts

```diff
@@ -0,0 +1,8 @@
+import { Network } from "../button/types";
+
+interface ILBLoaderAlt {
+  text: string;
+  network: Network;
+}
+
+export type { ILBLoaderAlt };
```

Comment:
recall, you don't have to export any type or interface you initialize inside types.ts unless you plan to use them in a file outside the same folder

Reply:
When you bring in a component or function from a different file—like how I imported Network from another file—your current file turns into a module. This means you need to explicitly export any interfaces or types if you want to use them in other files. If you don’t use imports, everything remains in a straightforward local scope, so you can use interfaces directly without exporting them.